packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -113,7 +116,9 @@ export default function CollabSidebar() {
 		reflowComments,
 		commentLastUpdated,
 	} = useBlockComments( postId );
-	useEnableFloatingSidebar( resultComments.length > 0 );
+	useEnableFloatingSidebar(
+		unresolvedSortedThreads.length > 0 || showCommentBoard
```

Comment:
Colocated the floating sidebar rendering logic here. The hook will close it automatically if it's no longer enabled.